manifests/node_exporter.pp

```diff
@@ -76,30 +85,32 @@
 #  [*version*]
 #  The binary release version
 class prometheus::node_exporter (
-  $arch                 = $::prometheus::params::arch,
-  $bin_dir              = $::prometheus::params::bin_dir,
-  $collectors           = $::prometheus::params::node_exporter_collectors,
-  $download_extension   = $::prometheus::params::node_exporter_download_extension,
-  $download_url         = undef,
-  $download_url_base    = $::prometheus::params::node_exporter_download_url_base,
-  $extra_groups         = $::prometheus::params::node_exporter_extra_groups,
-  $extra_options        = '',
-  $group                = $::prometheus::params::node_exporter_group,
-  $init_style           = $::prometheus::params::init_style,
-  $install_method       = $::prometheus::params::install_method,
-  $manage_group         = true,
-  $manage_service       = true,
-  $manage_user          = true,
-  $os                   = $::prometheus::params::os,
-  $package_ensure       = $::prometheus::params::node_exporter_package_ensure,
-  $package_name         = $::prometheus::params::node_exporter_package_name,
-  $purge_config_dir     = true,
-  $restart_on_change    = true,
-  $service_enable       = true,
-  $service_ensure       = 'running',
-  $service_name         = 'node_exporter',
-  $user                 = $::prometheus::params::node_exporter_user,
-  $version              = $::prometheus::params::node_exporter_version,
+  $arch                             = $::prometheus::params::arch,
```

Comment:
please add datatypes to those parameters. An example can be found at https://github.com/voxpupuli/puppet-prometheus/pull/68/files

Reply:
hm isn't adding datatypes to all params a different aspect that should be adressed in a seperate PR?